src/pybind/mgr/snap_schedule/module.py

```diff
@@ -70,7 +70,7 @@ def snap_schedule_get(self,
             return -errno.EINVAL, '', f"no such filesystem: {use_fs}"
         try:
             ret_scheds = self.client.get_snap_schedules(use_fs, path)
-        except CephfsConnectionException as e:
+        except Exception as e:
```

Comment:
If the exception is not a instance of `CephfsConnectionException`, the `e.to_tuple()` is going to throw an new exception, isn't it?

Reply:
I don't think a `return` statement would cause an exception to be raised
if an exception was supposed to be raised it should've been done with the `raise` statement

unless, I'm missing something you are trying to convey

Reply:
My point is - if we reach here with an exception that is not a subclass of CephfsConnectionException, the e.to_tuple() method would be invalid (since it does not exist for the exception instance).

Reply:
to_tuple() is a custom method that returns (-errno, str, str) tuple - would that be done exceptions that do not subclass from CephfsConnectionExcpetion?

Reply:
my mistake for not saying it right
the code could just be:
```
    ...
    ...
    except CephfsConnectionException as e:
        return e.to_tuple()
    except Exception as e:
        return -errno.EIO, '', str(e)
    ...
    ...
```